Dockerfile

```diff
@@ -0,0 +1,57 @@
+# Build the image from a ruby/alpine official image.
```

Comment:
Do a stage even before this one that only has a `FROM` statement. Then build `Builder` and `Final` from that one. That way you're not duplicating the `FROM`.